bgpd/bgp_packet.c

```diff
@@ -752,7 +752,7 @@ void bgp_notify_send_with_data(struct peer *peer, uint8_t code,
 	 */
 	if (peer->curr) {
 		size_t packetsize = stream_get_endp(peer->curr);
-		assert(packetsize <= sizeof(peer->last_reset_cause));
+		assert(packetsize <= sizeof(peer->max_packet_size));
```

Comment:
Isnt enough `assert(packetsize <= peer->max_packet_size);`? (dropped `sizeof()`).

Reply:
yep, that's a bug, nice eye